Scripts/assignment/emme_assignment.py

```diff
@@ -376,6 +376,38 @@ def aggregate_results(self,
                         miles["dist"][mode] += departures * segment.link.length
                         miles["time"][mode] += departures * segment[time_attr]
         resultdata.print_data(miles, "transit_kms.txt")
+        network = self.day_scenario.get_network()
+        attr_names = [attr.name for attr in self.day_scenario.extra_attributes()
+            if attr.type == "LINK"]
+        link_recs = ({
+            "geometry": {
+                "type": "LineString",
```

Comment:
This will loose the actual geometry of the link. We should be able to simply use LineString(link.shape) from shapely.geometry to get the required geometry instead of constructing it our self from the node coordinates.